eng/Version.Details.xml

```diff
@@ -146,22 +146,22 @@
       <Sha>aa59400b11e1aeee2e8af48928dbd48748a8bef9</Sha>
       <SourceBuild RepoName="vstest" ManagedOnly="true" />
     </Dependency>
-    <Dependency Name="Microsoft.NET.ILLink.Tasks" Version="8.0.20" CoherentParentDependency="Microsoft.NET.Sdk">
+    <Dependency Name="Microsoft.NET.ILLink.Tasks" Version="8.0.21" CoherentParentDependency="Microsoft.NET.Sdk">
       <Uri>https://dev.azure.com/dnceng/internal/_git/dotnet-runtime</Uri>
-      <Sha>574100b692e71fa3426931adf4c1ba42e4ee5213</Sha>
+      <Sha>362ab6669d55a75d51166f01b596c967c734ef4c</Sha>
     </Dependency>
     <Dependency Name="Microsoft.Net.Compilers.Toolset" Version="4.8.0-7.25056.18" CoherentParentDependency="Microsoft.NET.Sdk">
       <Uri>https://github.com/dotnet/roslyn</Uri>
       <Sha>a77b6e41d030878096fa496e00e4c0dffe5af9f0</Sha>
       <SourceBuild RepoName="roslyn" ManagedOnly="true" />
     </Dependency>
-    <Dependency Name="Microsoft.Build" Version="17.8.36" CoherentParentDependency="Microsoft.NET.Sdk">
-      <Uri>https://github.com/dotnet/msbuild</Uri>
-      <Sha>03e28d165c0959ee43aa91c2612776b011522a70</Sha>
+    <Dependency Name="Microsoft.Build" Version="17.8.43" CoherentParentDependency="Microsoft.NET.Sdk">
+      <Uri>https://dev.azure.com/devdiv/DevDiv/_git/DotNet-msbuild-Trusted</Uri>
```

Comment:
This doesn't look right and I suspect it is what is causing the VMR sync to fail.  IIRC the public builds know how to translate dev.azure.com/dnceng/internal to the equivalent GH url but I don't think there is logic for the trusted repos.  Let's try translating it to:

```suggestion
    <Dependency Name="Microsoft.Build" Version="17.8.43" CoherentParentDependency="Microsoft.NET.Sdk">
      <Uri>https://github.com/dotnet/msbuild</Uri>
```

This will require that the msbuild fixes were mirrored.